### PR TITLE
Update quanto blogpost

### PR DESCRIPTION
**Pull request by dacorvo**
This updates the quanto introduction blogpost to reflect its integration in the `optimum` collection of tools.

This also removes some lengthy explanations about the design that were partly obsolete.

It does not introduce yet the new `optimum`-like high-level API for `transformers` and `diffusers` models that will be presented in separate posts, some of them perhaps only under the `optimum` organization.

### CODE REVIEW

File: quanto-introduction.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: "Quanto: a pytorch quantization toolkit"
+title: "Quanto: a pytorch quantization backend for optimum"
```
Comment (stevhliu) [inline quanto-introduction.md:2]:
```suggestion
title: "Quanto: a PyTorch quantization backend for Optimum"
```

File: quanto-introduction.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 thumbnail: /blog/assets/169_quanto_intro/thumbnail.png
 authors:
 - user: dacorvo
 - user: ybelkada
 - user: marcsun13
 ---
 
-# Quanto: a pytorch quantization toolkit
+# Quanto: a pytorch quantization backend for optimum
```
Comment (stevhliu) [inline quanto-introduction.md:10]:
```suggestion
# Quanto: a PyTorch quantization backend for Optimum
```

File: quanto-introduction.md [hunk 3]
```diff
@@ -18,66 +18,66 @@ Many open-source libraries are available to quantize pytorch Deep Learning Model
 
 Also, although they are based on the same design principles, they are unfortunately often incompatible with one another.
 
-Today, we are excited to introduce [quanto](https://github.com/huggingface/quanto), a versatile pytorch quantization toolkit, that provides several unique features:
+Today, we are excited to introduce [quanto](https://github.com/huggingface/optimum-quanto), a pytorch quantization backend for [Optimum](https://huggingface.co/docs/optimum/index).
```
Comment (stevhliu) [inline quanto-introduction.md:21]:
```suggestion
Today, we are excited to introduce [quanto](https://github.com/huggingface/optimum-quanto), a PyTorch quantization backend for [Optimum](https://huggingface.co/docs/optimum/index).
```

File: quanto-introduction.md [hunk 4]
```diff
@@ -18,66 +18,66 @@ Many open-source libraries are available to quantize pytorch Deep Learning Model
...
 - automatically inserts quantization and dequantization stubs,
 - automatically inserts quantized functional operations,
 - automatically inserts quantized modules (see below the list of supported modules),
-- provides a seamless workflow for a float model, going from a dynamic to a static quantized model,
-- supports quantized model serialization as a `state_dict`,
-- supports not only `int8` weights, but also `int2` and `int4`,
-- supports not only `int8` activations, but also `float8`.
-
-Recent quantization methods appear to be focused on quantizing Large Language Models (LLMs), whereas [quanto](https://github.com/huggingface/quanto) intends to provide extremely simple quantization primitives for simple quantization schemes (linear quantization, per-group quantization) that are adaptable across any modality.
+- provides a seamless workflow from a float model to a dynamic to a static quantized model,
```
Comment (stevhliu) [inline quanto-introduction.md:30]:
```suggestion
- provides a seamless workflow from a float model to a dynamic and static quantized model,
```

File: quanto-introduction.md [hunk 5]
```diff
@@ -18,66 +18,66 @@ Many open-source libraries are available to quantize pytorch Deep Learning Model
...
+- serialization compatible with pytorch `weight_only` and 🤗 `safetensors`,
```
Comment (stevhliu) [inline quanto-introduction.md:31]:
```suggestion
- serialization compatible with PyTorch `weight_only` and 🤗 [Safetensors](https://huggingface.co/docs/safetensors/index),
```

File: quanto-introduction.md [hunk 6]
```diff
@@ -18,66 +18,66 @@ Many open-source libraries are available to quantize pytorch Deep Learning Model
...
+quantize(model, weights=qint8, activations=qint8)
 ```
 
-At this stage, the model's float weights are dynamically quantized only for inference.
+At this stage, only the inference of the model is modified to dynamically quantize the weights.
 
 **2. Calibrate (optional if activations are not quantized)**
 
-Quanto supports a calibration mode that records the activation ranges while passing representative samples through the quantized model.
+Quanto supports a calibration mode that allows to record the activation ranges while passing representative samples through the quantized model.
```
Comment (stevhliu) [inline quanto-introduction.md:62]:
```suggestion
Quanto supports a calibration mode that allows the recording of the activation ranges while passing representative samples through the quantized model.
```

File: quanto-introduction.md [hunk 7]
```diff
@@ -90,32 +90,75 @@ for batch_idx, (data, target) in enumerate(train_loader):
...
+Both `pickle` and `safetensors` (recommended) are supported.
+
+```python
+from safetensors.torch import save_file
+
+save_file(model.state_dict(), 'model.safetensors')
+```
+
+In order to be able to reload these weights, you also need to store the quantized
+model quantization map.
```
Comment (stevhliu) [inline quanto-introduction.md:113]:
```suggestion
In order to reload these weights, you also need to store the quantized
models quantization map.
```

File: quanto-introduction.md [hunk 8]
```diff
@@ -90,32 +90,75 @@ for batch_idx, (data, target) in enumerate(train_loader):
...
+from optimum.quanto import quantization_map
+
+with open('quantization_map.json', w) as f:
+  json.dump(quantization_map(model))
+```
+
+**5. Reload a quantized model**
+
+A serialized quantized model can be reloaded from a `state_dict` and a `quantization_map` using the `requantize` helper.
+Note that you need first to instantiate an empty model.
```
Comment (stevhliu) [inline quanto-introduction.md:127]:
```suggestion
Note that you need to first instantiate an empty model.
```

**Review by stevhliu — APPROVED** [review]
LGTM, thanks for the update! 🙂

File: quanto-introduction.md (hunk 4 above)
Comment (dacorvo) [reply quanto-introduction.md:30]: actually it is dynamic, then static

**Pull request merged**